docs/source/support/release-notes.rst

```diff
@@ -8,6 +8,7 @@ This page contains release notes for the SDK.
 
 HEAD — ongoing
 --------------
+- Enabling support for Postgres persistence under the Sandbox
```

Comment:
I think this could help the user more in understanding what's changed. Maybe: "Adds support for using a Postgres database as a back end for the Sandbox, which gives you persistent data storage."

Plus it's always good to give people a next step. For example, "For information on how to start using this feature, see <link>."

Reply:
good idea, i changed it